access/src/main/java/dk/dbc/holdingsitems/RecordCollection.java

```diff
@@ -49,22 +49,6 @@
     boolean original;
     private final String trackingId;
 
-    /**
-     * From database, sets original false
```

Comment:
Denne kommentar virker vigtig.
Min første tanke er, at "original" har en betydning som jeg skal notere mig, og at det er vigtigt at benytte de to constructorer rigtigt.
-- At de så måske burde have været en FactoryMethods med et bedre navn er en anden sag.